src/go/configinfo/service_info.go

```diff
@@ -719,6 +719,17 @@ func (s *ServiceInfo) processLocalBackendOperations() error {
 }
 
 func (s *ServiceInfo) processUsageRule() error {
+
+	// TODO(dafudeng) b/238652680
```

Comment:
Remove TODO?

Reply:
Yes, planned to add integration test and will do it. 
I put TODO here for two reasons: 1. it does not look elegant 2. if user wants to define their own health check mechanism, then we may need to implement a more generic way to allow the health check to bypass service control, i.e. use flag --healthcheck=grpc.health.v1.Health. If a more generic implementation is available, this hardcoded logic will be replaced.

Let me know if the above make sense and if they are real concerns, especially the second one.

Reply:
I have two concerns
1) chnages here also impact ESPv2.  It may break existing customers,  should we do it here?
2) Users may want to "skip_service_control" their custom health service, do we want to support it.

Reply:
> 1. chnages here also impact ESPv2. It may break existing customers, should we do it here?

This is not rejecting the config for ESPv2 customers, it is just adding in the skip_service_control config. It shouldn't break any customers, unless some customer defined gRPC health check protocol and doesn't want to skip service control for it. I doubt that is the case.

> 2. if user wants to define their own health check mechanism

I don't think 2 is a concern. If customer is using gRPC backend, they should be using gRPC health check protocol.

For ESPv2, we still allow customers to define their own mechanism. We don't reject any configs with skip_service_control. So there is no issue.

We can enforce that Cloud ESF users follow gRPC health check protocol.

Reply:
SG

Reply:
TODO removed.